src/Playwright.CLI/Playwright.CLI.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageOutputPath>./nupkg</PackageOutputPath>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <IsPackable>true</IsPackable>
-    <RollForward>Major</RollForward>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
Documenting this: 

 This will make it run on the latest runtime it can find (excluding pre-release), not the lowest. So you will target netcoreapp3.1 so anyone who has netcoreapp3.1 only is able to run your app. But most people have newer version and they will run using the newer version.